docs/hub/spaces-dependencies.md

```diff
@@ -12,7 +12,8 @@ The default Spaces environment comes with several pre-installed dependencies:
 
 * [`datasets`](https://github.com/huggingface/datasets) allows you to fetch or display any dataset from the Hub inside your app.
 
-* The SDK you specified, which could be either `streamlit` or `gradio`. The version is specified in the `README.md` file.
+* Gradio version is specified by the `sdk_version` in the `README.md` file.
```

Comment:
```suggestion
* [`gradio`](https://github.com/gradio-app/gradio), if you selected it as your Space type. You can optionally require a specific version using [`sdk_version` in the `README.md` file](spaces-config-reference).
```

Reply:
As streamlit is removed from the default SDK options, this section now applies only to Gradio. So I'm not sure if this change is needed.